sdk/containerapps/Azure.ResourceManager.AppContainers/src/Generated/Models/ContainerAppJobExecutionBase.cs

```diff
@@ -11,7 +11,7 @@ namespace Azure.ResourceManager.AppContainers.Models
     public partial class ContainerAppJobExecutionBase
     {
         /// <summary> Initializes a new instance of ContainerAppJobExecutionBase. </summary>
-        public ContainerAppJobExecutionBase()
+        internal ContainerAppJobExecutionBase()
```

Comment:
Breaking change. Looks like this model is changed to abstract?

Reply:
This was not used - `JobExecutionNamesCollection` and was removed.

Reply:
`internal` is the correct modifier here.